masterlist.yaml

```diff
@@ -13296,20 +13294,15 @@ plugins:
         condition: 'active("Complete Alchemy & Cooking Overhaul.esp") and not active("CACO_Wyrmstooth_Patch.esp")'
       - <<: *obsolete
         subs: [ '[Wyrmstooth](https://www.nexusmods.com/skyrimspecialedition/mods/45565/)' ]
-        condition: 'not version("Wyrmstooth.esp", "1.19", >=)'
-    dirty:
-      - <<: *quickClean
-        crc: 0xF73A6161
-        util: '[SSEEdit v4.0.3](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
-        itm: 5587
-    clean:
+        condition: 'not version("Wyrmstooth.esp", "1.19.2", >=)'
+    clean: # CRC's for 1: Unknown, 2: Unknown, 3: Drachenzahn (Wyrmstooth) DV 1.19.2, 4: Wyrmstooth 1.19.2 #
```

Comment:
We can put those in the commit message like below.
![image](https://user-images.githubusercontent.com/1944639/113022731-d32f5980-917c-11eb-86a5-e5f5409c8f9f.png)


> CRC's for 1: Unknown, 2: Unknown, 3: Drachenzahn (Wyrmstooth) DV 1.19.2, 4: Wyrmstooth 1.19.2 #

```suggestion
    clean:
```

Reply:
I'd prefer keeping something like that so when people update the crc's they know which one to update

Reply:
I'll remove it then